### PR TITLE
Update black and remove lock file

### PR DESCRIPTION
**Pull request by Mgancita**

### CODE REVIEW

File: {{cookiecutter.project_slug}}/poetry.lock [hunk 1]
```diff
@@ -1,917 +0,0 @@
-[[package]]
```
Comment (MaKaNu) [inline {{cookiecutter.project_slug}}/poetry.lock:1]: Why did you removed the complete lock file? I 've red somewhere in the docs of poetry, that the lock is used for something if it is on the repo. 

Found it: (https://python-poetry.org/docs/basic-usage/#installing-with-poetrylock)[https://python-poetry.org/docs/basic-usage/#installing-with-poetrylock]

Comment (Mgancita) [reply {{cookiecutter.project_slug}}/poetry.lock:1]: oh I was testing updating it and must have left it removed. I'll update it now

Comment (Mgancita) [reply {{cookiecutter.project_slug}}/poetry.lock:1]: alright so I initially was right with removing it. The reason was we don't know what the user is going to need until they select the cookiecutter options (flake8, black, etc.). With the first command of `poetry install`, it'll setup the `poetry.lock` file for the user which works better.

Comment (MaKaNu) [reply {{cookiecutter.project_slug}}/poetry.lock:1]: This sounds kind of logical the lock file is necessary for final product not for the development template.

**Review by MaKaNu — APPROVED** [review]

**Pull request merged**